src/components/AddPlaylist/style.module.css

```diff
@@ -0,0 +1,50 @@
+.modalCont {
+  position: fixed;
+  left: 0;
+  top: 0;
+  width: 100%;
+  height: 100%;
+  overflow: auto;
+  display: flex;
+  justify-content: center;
+  align-items: center;
+  background-color: rgb(0, 0, 0);
+  background-color: rgba(0, 0, 0, 0.4);
+}
+
+.modal {
+  width: 300px;
```

Comment:
Can Use rem Values here.

Reply:
yes we can use but I want my modal to be fixed for this particular use case So I put it in px's